web/client-api/src/main/java/io/deephaven/web/client/api/widget/JsWidget.java

```diff
@@ -72,14 +121,76 @@ public Uint8Array getDataAsU8() {
         return response.getData_asU8();
     }
 
+    @JsMethod
+    public String getDataAsString() {
+        return new String(Js.<byte[]>uncheckedCast(response.getData_asU8()), StandardCharsets.UTF_8);
+    }
+
     @JsProperty
     public JsWidgetExportedObject[] getExportedObjects() {
-        if (this.exportedObjects == null) {
-            this.exportedObjects = response.getTypedExportIdList().asList().stream()
-                    .map(ticket -> new JsWidgetExportedObject(connection, ticket))
-                    .toArray(JsWidgetExportedObject[]::new);
+        return Js.<JsWidgetExportedObject[]>uncheckedCast(exportedObjects);
+    }
+
+    @JsMethod
+    public JsWidgetExportedObject getExportedObject(int index) {
```

Comment:
@mattrunyon does this serve a specific purpose outside of just using `object.exportedObjects[index]`?

Reply:
Nothing I can think of or remember